devdox/tests/test_repos.py

```
@@ -10,8 +10,7 @@
 from datetime import datetime
 
 from app.utils import constants
-from app.config import GitHosting
-from app.schemas.repo import GitHostingProvider
+from devdox.app import GitHosting
```

Pylint (3.3.7)

[error] No name 'GitHosting' in module 'devdox.app'

(E0611)